src/status_im/subs/wallet/wallet.cljs

```diff
@@ -38,9 +38,24 @@
  :-> :scanned-address)
 
 (rf/reg-sub
- :wallet/tokens-loading?
+ :wallet/tokens-loading
  :<- [:wallet/ui]
- :-> :tokens-loading?)
+ :-> :tokens-loading)
+
+(rf/reg-sub
+ :wallet/home-tokens-loading?
+ :<- [:wallet/tokens-loading]
+ (fn [tokens-loading]
+   (->> tokens-loading
+        vals
+        (some true?))))
+
+(rf/reg-sub
+ :wallet/current-viewing-account-tokens-loading?
+ :<- [:wallet/tokens-loading]
+ :<- [:wallet/current-viewing-account-address]
+ (fn [[tokens-loading current-viewing-account-address]]
+   (get tokens-loading current-viewing-account-address)))
```

Comment:
Made the tokens loading flag granular to the account

Reply:
Let's be careful with this change, it may create bugs. Is there a special reason to change this?

Since we aren't able to see two accounts at the same time, I wonder if makes sense to know that an account that is off screen is loaded or loading :thinking: 

I think this change needs some extra refactors, such as when we create a new account, make sure to properly mark it this key as loading, same when they are received. 

I'd like to know the reason to make this change because we may be adding some extra complexity that maybe we don't need.

Reply:
>  Is there a special reason to change this?

When we receive a backed-up watch-only account or keypair from Waku, we process the accounts individually (by storing them in our app-db, fetching assets, and balances) as we can't rely on a single boolean/flag (`tokens-loading?`) as it will affect existing account(s). So, I made it granular so that each account can refer to its token loading state (in the account screen and account card at home). For the home tab - aggregated tokens and balance - if any accounts are loading, it will show the loading UI. 

Additionally, we won't receive all data at once. So, we need to make it a granular approach for easy processing.

Also, it's not good to call the `wallet_getAccounts` (or any assets RPC) every time we receive any backed-up data as it will make a lot of requests/processing time.

> Since we aren't able to see two accounts at the same time, I wonder if makes sense to know that an account that is off screen is loaded or loading 🤔

The existing flag `tokens-loading?` is global and shows loading for all account cards as it fetches all accounts in one shot. Making it granular at least shows balances for accounts that are loaded while the user can scroll through it.

> I think this change needs some extra refactors, such as when we create a new account, make sure to properly mark it this key as loading, same when they are received.

When a new account is added, we fetch all accounts and other chain of events (tokens, collectibles,..etc) that are dispatched as usual, It's the existing feature/code. It ensures the key is marked as loading (for every account).

Reply:
@smohamedjavid Thanks for the explanation.

Again, just make sure to properly update the code that is related and that may be affected.

Will we ask for manual QA to make sure nothing is broken?

Reply:
> Will we ask for manual QA to make sure nothing is broken?

Yes, we will do a manual QA for this PR 👍